wiki/osu!tourney/en.md

```diff
@@ -332,34 +327,33 @@ Rename or delete the `tournament.cfg` file.
 
 ### My client is not spectating!
 
-**Make sure you have joined the room!**
+Make sure you have **joined the room!**
 
 Click the room name on the bottom black panel until it is **bold**.
 If the client is still not spectating, click the `Panic` button.
 
 ### The rooms are not showing up!
 
-- Your multiplayer room is named incorrectly or you used the wrong acronym in the `tournament.cfg`.
-- Make sure you use the correct room name (see the **Match Creation** section).
-  Renaming an existing multiplayer room will not work.
-- Recreate the entire room, if you did something wrong.
+It is possible that your multiplayer room is named incorrectly or you used the wrong acronym in the `tournament.cfg`.
+Make sure you are using the correct room name (see the **Match Creation** section), renaming an existing multiplayer room will not work.
+Recreate the entire room if you did something wrong.
 
 ### The team names are not showing!
 
-- Your multiplayer room is named incorrectly then or you use the wrong acronym in the `tournament.cfg`.
-- Make sure you use the correct room name (see the **Match Creation** section).
-  Renaming an existing multiplayer room will not work.
-- Recreate the entire room, if you did something wrong.
+It is possible that your multiplayer room is named incorrectly or you used the wrong acronym in the `tournament.cfg`.
+Make sure you use the correct room name (see the **Match Creation** section), renaming an existing multiplayer room will not work.
+Recreate the entire room if you did something wrong.
+
+Post in [this thread](https://osu.ppy.sh/forum/t/327120) if you need further assistance.
 
 ### How do I stream my tournament to streaming services like Twitch?
 
 To set up a stream, we recommend using the [Open Broadcaster Software](https://obsproject.com/), but know that you are also free to use any other streaming client you have access to.
-The resolution of the top part of osu!tourney is `1280x720px`.
-
-**Please adjust the cropping to remove the black control panel from the stream!**
+The resolution of the top part of osu!tourney can be customised but by default it is `1280 × 720`. Make sure yoir monitor is bigger than this value. See the Setup section for information on how to change the resolution.
+Do not forget to adjust the cropping to remove the black control panel from the stream!
 
 ### My issue/question is not listed here! What to do?
 
 Post in **[this thread](https://osu.ppy.sh/forum/t/327120)**.
```

Comment:
This needs to be removed and should link back to `tournaments@ppy.sh`. The line below this one also, but keep the screenshots one.